cmd/soroban-cli/src/config/secret.rs

```diff
@@ -43,29 +43,30 @@ pub struct Args {
 
 impl Args {
     pub fn read_secret(&self) -> Result<Secret, Error> {
+        let print = Print::new(false);
         if let Ok(secret_key) = std::env::var("SOROBAN_SECRET_KEY") {
+            print.infoln("Read secret key from environment variable SOROBAN_SECRET_KEY");
             Ok(Secret::SecretKey { secret_key })
```

Comment:
Just realized there is no check here that it is a valid secret key. That's what we get for using a `String` when we should just use `PrivateKey` directly.

Reply:
Yup, see:
- https://github.com/stellar/stellar-cli/pull/1812